### ISSUE TITLE
'new NativeEventEmitter()' was called with a non-null argument without the required 'addListener/removeListeners' method.

### ISSUE THREAD

**Issue by Bej-Technologies** [Leanplum/Leanplum-ReactNative-SDK#60]
I am getting the following warnings when running my app on Android with `"@leanplum/react-native-sdk": "^1.2.1"` on `"react-native": "0.65.0"`.

```
'new NativeEventEmitter()' was called with a non-null argument without the required 'addListener' method.
'new NativeEventEmitter()' was called with a non-null argument without the required 'removeListeners' method.
```

It seems to be solved by adding the following to `LeanplumInboxModule.java` and `LeanplumModule.java`:

```
@ReactMethod
public void addListener(String eventName) {
    // Needed to remove warning
}

@ReactMethod
public void removeListeners(Integer count) {
    // Needed to remove warning
}
```

It seems to be related to this change: https://github.com/facebook/react-native/commit/114be1d2170bae2d29da749c07b45acf931e51e2

**Comment by hborisoff** [comment]
@Bej-Technologies Thank you for noting that!
I tried it and it solves the warning. We'll include it in the next release.

**Comment by nzagorchev** [comment]
Fixed in 1.3.0
